cylc/flow/cfgspec/globalcfg.py

```diff
@@ -745,13 +744,15 @@
             Conf('job submission environment pass-through', VDR.V_STRING_LIST,
                  desc='''
                 Minimal list of environment variable names to pass through to
-                job submission subprocesses. $HOME is passed automatically.
+                job submission subprocesses. ``$HOME`` is passed automatically.
                 You are unlikely to need this.
             ''')
-            Conf('job submission executable paths', VDR.V_STRING_LIST, desc='''
+            Conf('job submission executable paths', VDR.V_STRING_LIST,
+                 desc=f'''
                 Additional executable locations to pass to the job
-                submission subprocess beyond the standard locations''' +
-                 ', '.join(SYSPATH) + '''. You are unlikely to need this.
+                submission subprocess beyond the standard locations
+                {", ".join(f"``{i}``" for i in SYSPATH)}.
+                You are unlikely to need this.
```

Comment:
Wouldn't it be better to explain why you _would_ need this instead of just saying you probably won't need this?